tests/test_config.py

```diff
@@ -50,16 +52,11 @@ def test_config_env_access_error(config):
         _ = config.DOES_NOT_EXIST
 
 
-def test_config_env_access_raise_error_if_network_config_not_json(monkeypatch, config):
-    monkeypatch.setenv("ALMA_SAP_INVOICES_ECS_NETWORK_CONFIG", "test")
-    with pytest.raises(InvalidNetworkConfigurationError, match="Expecting value"):
-        _ = config.ALMA_SAP_INVOICES_ECS_NETWORK_CONFIG
-
-
-def test_config_env_access_raise_error_if_network_config_is_missing(monkeypatch, config):
-    monkeypatch.delenv("ALMA_SAP_INVOICES_ECS_NETWORK_CONFIG")
-    with pytest.raises(
-        InvalidNetworkConfigurationError,
-        match="the JSON object must be str, bytes or bytearray, not NoneType",
-    ):
-        _ = config.ALMA_SAP_INVOICES_ECS_NETWORK_CONFIG
+def test_config_env_access_network_config_success(monkeypatch, config):
+    assert config.ALMA_SAP_INVOICES_ECS_NETWORK_CONFIG == {
+        "awsvpcConfiguration": {
+            "subnets": ["subnet-abc123"],
+            "securityGroups": ["sg-abc123"],
```

Comment:
related to my comment on the preceding line, it is very unlikely that the "securityGroups" value will ever have anything more than just a single security group.

Reply:
See response in https://github.com/MITLibraries/alma-sapinvoices-ui/pull/83#discussion_r1742667164. 😄